src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,97 @@
+package main.java.Duke;
+
+import main.java.Duke.Task.Deadline;
+import main.java.Duke.Task.Event;
+import main.java.Duke.Task.Task;
+import main.java.Duke.Task.toDo;
+
+import java.io.*;
```

Comment:
Instead of using the wildcard(*), do list out all the import statements used from the java.io package.